app/experimenter/experiments/email.py

```diff
@@ -14,96 +14,75 @@ def send_intent_to_ship_email(experiment_id):
     percent_of_population = f"{float(experiment.population_percent):g}%"
 
-    content = render_to_string(
-        "experiments/emails/intent_to_ship.txt",
+    format_and_send_html_email(
+        experiment,
+        "experiments/emails/intent_to_ship.html",
         {
             "experiment": experiment,
             "bug_url": bug_url,
             "percent_of_population": percent_of_population,
             "locales": [str(l) for l in experiment.locales.all()],
             "countries": [str(c) for c in experiment.countries.all()],
         },
-    )
-    # Strip extra newlines from autoescape tag
-    content = content.strip() + "\n"
-
-    channel = experiment.firefox_channel
-    email = EmailMessage(
-        Experiment.INTENT_TO_SHIP_EMAIL_SUBJECT.format(
-            name=experiment.name,
-            version=experiment.format_firefox_versions,
-            channel=channel,
-        ),
-        content,
-        settings.EMAIL_SENDER,
-        [settings.EMAIL_RELEASE_DRIVERS],
-        cc=[experiment.owner.email],
-    )
-    email.send(fail_silently=False)
-
-    ExperimentEmail.objects.create(
-        experiment=experiment, type=Experiment.INTENT_TO_SHIP_EMAIL_LABEL
+        Experiment.INTENT_TO_SHIP_EMAIL_SUBJECT,
+        Experiment.INTENT_TO_SHIP_EMAIL_LABEL,
+        cc_recipients=[settings.EMAIL_RELEASE_DRIVERS],
     )
 
 
 def send_experiment_launch_email(experiment):
-
-    content = render_to_string(
-        "experiments/emails/launch_experiment_email.txt",
-        {"experiment": experiment},
-    )
-
-    version = experiment.format_firefox_versions
-    channel = experiment.firefox_channel
-
-    recipients = [experiment.owner.email] + list(
-        experiment.subscribers.values_list("email", flat=True)
-    )
-
-    email = EmailMessage(
-        Experiment.LAUNCH_EMAIL_SUBJECT.format(
-            name=experiment.name, version=version, channel=channel
-        ),
-        content,
-        settings.EMAIL_SENDER,
-        recipients,
-    )
-
-    email.send(fail_silently=False)
-
-    ExperimentEmail.objects.create(
-        experiment=experiment, type=Experiment.EXPERIMENT_STARTS
+    format_and_send_html_email(
+        experiment,
+        "experiments/emails/launch_experiment_email.html",
+        {
+            "experiment": experiment,
+            "change_window_url": ExperimentConstants.NORMANDY_CHANGE_WINDOW,
+        },
+        Experiment.LAUNCH_EMAIL_SUBJECT,
+        Experiment.EXPERIMENT_STARTS,
     )
 
 
 def send_experiment_ending_email(experiment):
 
-    html_content = render_to_string(
+    format_and_send_html_email(
+        experiment,
         "experiments/emails/experiment_ending_email.html",
         {
             "experiment": experiment,
             "change_window_url": ExperimentConstants.NORMANDY_CHANGE_WINDOW,
         },
+        Experiment.ENDING_EMAIL_SUBJECT,
+        Experiment.EXPERIMENT_ENDS,
     )
 
+
+def format_and_send_html_email(
```

Comment:
So we don't test this function directly, we just test its uses in the specific emails?

Reply:
Yes I see what you mean, that would have prevent some small duplication across all tests... maybe for the next refactor 🤓